doc/HOWTO_RELEASE.rst.txt

```diff
@@ -34,24 +36,39 @@ Release Scripts
 ---------------
 * https://github.com/numpy/numpy-vendor
 
-
 Supported platforms and versions
 ================================
-Python 2.6-2.7 and >=3.2 are the currently supported versions on all platforms.
+
+Python 2.6-2.7 and >=3.2 are the currently supported versions when building
+from source.  We test numpy against all these versions every time we merge
+code to trunk.  Binary installers may be available for a subset of these
+versions (see below).
 
 OS X
 ----
-OS X versions >= 10.5 are supported.  Note that there are currently still
-issues with compiling on 10.7, due to Apple moving to gcc-llvm.
-Only the Python from `python.org <http://python.org>`_ is supported. Binaries
-do *not* support Apple Python.
+
+Python 2.7 and >=3.3 are the versions for which we provide binary installers.
+
+OS X versions >= 10.6 are supported.  We build binary wheels for OSX that are
+compatible with Python.org Python, system Python, homebrew and macports - see
+this `OSX wheel building summary
+<https://github.com/MacPython/wiki/wiki/Spinning-wheels>`_ for details.
 
 Windows
 -------
+
+32-bit Python 2.7, 3.3, 3.4 are the versions for which we provide binary
+installers.
+
 Windows XP, Vista and 7 are supported.
 
```

Comment:
Might remove this blank line and merge above an below parts, similar to the OS X documentation.